Manager/ManagerInterface.php

```diff
@@ -31,20 +31,31 @@ public function create($entityClass, array $defaultValues = array(), array $opti
      *
      * @return object
      */
-    public function find($entityClass, $id, array $options = array());
+    public function find($entityClass, $id, array $options = []);
 
     /**
      * Saves the entity
      *
      * @param object $entity
      * @param array  $options
      */
-    public function save($entity, array $options = array());
+    public function save($entity, array $options = []): void;
 
     /**
      * Remove the entity
      *
      * @param object $entity
      */
-    public function remove($entity);
+    public function remove($entity): void;
+
+    /**
+     * Normalize custom entity object to array
+     *
+     * @param       $entity
+     * @param null  $format
+     * @param array $context
+     *
+     * @return array
+     */
+    public function normalize($entity, $format = null, array $context = []): array;
```

Comment:
entiyt type => object

Reply:
i tried to and had some issue

Reply:
which one? I thought only the type of object was enforced but the variable name was free. Maybe some PHP 7.X change.

Reply:
`object` typehint is for PHP 7.2 https://wiki.php.net/rfc/object-typehint